client/src/components/ProfileForm.js

```diff
@@ -23,26 +49,80 @@ const TableUserAttributes = ({ userAttributes }) => (
 			      hideOnScroll
     			/>
     		</Table.HeaderCell>
-				<Table.HeaderCell width={3}>Current Value</Table.HeaderCell>
-				<Table.HeaderCell width={5}>New Value</Table.HeaderCell>
+				<Table.HeaderCell width={3}>
+					Current Value
+				</Table.HeaderCell>
+				<Table.HeaderCell width={5}>
+					New Value
+				</Table.HeaderCell>
 			</Table.Row>
 		</Table.Header>
 		<Table.Body>
-			{userAttributes.map(userAttributes => {
-				const { name,value,type } = {...userAttributes}
+			{userAttributes.map((userAttributes, index) => {
+				const { name, type } = {...userAttributes}
+				let fieldName = []
```

Comment:
fieldName can't be an array. You can remove the let fieldName = [].
Append the let to the fieldName = 'user.userAttributes...' on line 65.
Remove the if (type === 'string'). Otherwise, email will not have a fieldName, and the initial value will not show up in that field.